data/human/human missions.txt

```diff
@@ -3601,3 +3601,79 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
+	# Don't require the player to pay credits again if the player was already allowed to land when this mission triggered.
+	clearance
+	to offer
+		random < 8
+		credits >= 2000000
+	on offer
+		conversation
+			"You see a warning that one of the repulsors on your ship is not working when you start de-orbiting."
+			choice
+				"(Return to orbit to fix the issue in space.)"
+					launch
+				"(Attempt to land anyway.)"
+			"Orbital entry is a bit more steep than normal and your ship is starting to lean a bit."
+			choice
+				"(Try to return to orbit to still fix the issue in space.)"
```

Comment:
```suggestion
				"(Try to return to orbit to fix the issue in space.)"
```
I don't think the usage of `still` really makes sense here. Were you just looking for some way to distinguish it from previous option?

Reply:
It indeed looks better without the word still. I guess I initially added it because the player already choose not to return to space multiple times when he/she gets to this point.